### ISSUE TITLE
Rostock profile?

### ISSUE THREAD

**Issue by sacsac345** [spegelius/filaswitch#69]
Does anyone have a Rostock hw profile and Simplify3d scripts they'd be willing to share?

thanks

**Comment by Ginko72** [comment]
Yes, I do.  This only works for S3D 4.1.2 (5.x not supported at this time).  There are a number of macros that were originally described in a FRM file, but I just put those on my printer as Duet Macros.  Here are the links:

FFF File - https://drive.google.com/file/d/13Ap6BueeqFYB6ZhF047cZhdzw-kxqRQI/view?usp=drive_link
Dual Extrusion Macros for Duet - https://drive.google.com/file/d/1kW3FlmChYWwvCAV24NMAHOuKM_Jp0XVF/view?usp=drive_link
